scripts/biomedical/chembl/test-data/chembl_29_fasta_expected.csv

```diff
@@ -0,0 +1,11 @@
+Sequence,chembl,uniprot,name,dcid
```

Comment:
underscores should be replaced with spaces from the name. Again text values are not properly indicated.